vmdb/app/models/ems_event/parsers/kubernetes.rb

```diff
@@ -0,0 +1,21 @@
+module EmsEvent::Parsers::Kubernetes
+  def self.event_to_hash(event, ems_id = nil)
+    log_header = "MIQ(#{name}.event_to_hash) ems_id: [#{ems_id}]"
+
+    $log.debug("#{log_header} event: [#{event.inspect}]")
+
+    event_type = "#{event[:kind].upcase}_#{event[:reason].upcase}"
```

Comment:
I sorta think the `event_type` here could be prefixed with `"KUBE"` or something similar.  We already have event types for RHEVM that have all uppercase with underscores.  It would be nice to have a distinction for the Kubernetes events.

@Fryguy, thoughts on this?